fusor-ember-cli/app/templates/components/text-f.hbs

```diff
@@ -3,7 +3,7 @@
   {{input class="form-control" value=value placeholder=placeholder type=typeInput focus-out="showValidationErrors" id=cssId disabled=disabled autocomplete='off'
   maxlength="250"}}
 
-  {{#if isPassword}}
+  {{#if canShowPassword}}
           <i {{action 'showPassword'}} class="fa {{eyeIcon}} eye-icon"></i>
   {{/if}}
```

Comment:
These 3 lines could be deleted since `canPassword` is not defined anyway so it's always false. I guess it doesn't hurt to leave it in case UX wants to add back this eye open/close functionality.